include/openssl/ec.h

```diff
@@ -422,7 +447,8 @@ size_t EC_get_builtin_curves(EC_builtin_curve *r, size_t nitems);
 
 const char *EC_curve_nid2nist(int nid);
 int EC_curve_nist2nid(const char *name);
-int EC_GROUP_check_named_curve(const EC_GROUP *group, int nist_only);
+int EC_GROUP_check_named_curve(const EC_GROUP *group, int nist_only,
```

Comment:
You seem to add new functions with the `_ex` suffix, which is commendable considering we want to avoid API breaks,... except here. Any reason why?

Reply:
I believe it is because `EC_GROUP_check_named_curve` is new functionality only existing in `master`

Reply:
Ah, ok

Reply:
Yes - exactly.